src/addNavigationHelpers.js

```diff
@@ -61,6 +61,12 @@ export default function(navigation) {
       return navigation.dispatch(NavigationActions.setParams({ params, key }));
     },
 
+    getParam: (paramName, defaultValue) => {
+      const param =
+        navigation.state.params && navigation.state.params[paramName];
+      return param || defaultValue;
```

Comment:
You should have an `if` condition testing for the presence of the `paramName` in `params` instead of doing an or. In the current implementation if params is `params: {foo: 0}` then `getParam('foo')` will return `undefined` instead of `0`.

Reply:
:+1: thanks!